apps/common/views/partials/maincontent-left.html

```diff
@@ -0,0 +1,9 @@
+{{$content-outer}}
+  <div class="govuk-grid-column-two-thirds">
+    <header>
+      {{#header}}<h1>{{$header}}{{/header}}</h1>{{/header}}
+    </header>
+    {{$validationSummary}}{{/validationSummary}}
+    {{$content}}{{/content}}
+  </div>
+{{/content-outer}}
```

Comment:
Could you explain why you needed this?

Reply:
This was done to change the font size to gov-uk design system’s recommendedation - without it the header defaults to xtra large instead of the recommended large.